main/client/sectionHelloFriends/sectionHelloFriends.html

```diff
@@ -0,0 +1,18 @@
+<template name="sectionHelloFriends">
+    <section id="lets-be-friends">
+        <div id="children-pics">
+            <img src="images/gfx_10.jpg">
```

Comment:
add "col-lg-3" class to each img element (lets decide later on mobile adaptation)